usmqe_tests/web/others/test_lists.py

```diff
@@ -63,7 +63,7 @@ def test_cluster_list(valid_credentials, sds_name):
     NOTE: a 'gluster' cluster must already exists in etcd
     """
     navMenuBar = valid_credentials.init_object
```

Comment:
please rename the variable as it is not Navigation menu any more, but cluster list